cmd/senpai/main.go

```diff
@@ -87,11 +83,11 @@ func getLastBufferPath() string {
 	return lastBufferPath
 }
 
-func getLastBuffer() (string, error) {
+func getLastBuffer() string {
 	buf, err := ioutil.ReadFile(getLastBufferPath())
 	if err != nil {
-		return "", fmt.Errorf("failed to read last buffer: %s\nignore this if your running senpai for the first timeee~", err)
```

Comment:
Is it okay no to handle error here? Since it will always fail on first run.

Reply:
Right, we don't want to print an error on the first run